src/SprykerEco/Zed/Econda/Dependency/Plugin/EcondaPluginInterface.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Econda\Dependency\Plugin;
+
+use Generated\Shared\Transfer\BatchResultTransfer;
+use Generated\Shared\Transfer\LocaleTransfer;
+use SprykerEco\Zed\Econda\Business\Exporter\Writer\WriterInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+interface EcondaPluginInterface
+{
+    /**
+     * @param \Generated\Shared\Transfer\LocaleTransfer $locale
+     * @param \Generated\Shared\Transfer\BatchResultTransfer $result
```

Comment:
Naming: `$localeTransfer` and `$batchResultTransfer`

Reply:
Done